terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/string/StringController.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (C) 2013-2016 terasoluna.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.gfw.functionaltest.app.string;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.terasoluna.gfw.common.fullhalf.DefaultFullHalf;
+import org.terasoluna.gfw.common.fullhalf.FullHalfPairsBuilder;
+
+@Controller
+@RequestMapping(value = "string")
+public class StringController {
+	
+	@RequestMapping(method = RequestMethod.GET)
+	public String index() {
+		
+		return "string/index";
+	}
+	
+	@RequestMapping(value="01", method = RequestMethod.GET)
+	public String string01(Model model) {
+		
+		model.addAttribute("string", DefaultFullHalf.INSTANCE.toFullwidth("ｱﾞ!A8ｶﾞザ"));
```

Comment:
@btsatoukzn @ikeyat 
It is unnatural to implement the test input values to the controller.
I think test input values should be implemented in testcase.
(If necessary , it is also possible to review in the internal.)